controller/controller.js

```diff
@@ -35,13 +36,20 @@ function getCommentsByReviewId(request, response, next) {
     }).catch(next)
 }
 
+
 function postCommentByReviewId(request, response, next) {
     const {review_id} = request.params
     Promise.all([fetchReviewsByReviewId(review_id), addCommentByReviewId(review_id, request.body)]).then((result) => {
         response.status(201).send({comment: result[1]})
     }).catch(next)
 
+function patchReview(request, response, next) {
+    const {review_id} = request.params
+    const {inc_votes} = request.body
+    updateReview(review_id, inc_votes).then((result) => {
+        response.status(202).send(result)
```

Comment:
Would be a 200, and it's also best practise to send response back as an object, with a key of whatever you're sending back eg {updatedReview: {...the updated review}}